Misc/Regions_Park Air MAN.txt

```diff
@@ -1,7 +1,9 @@
-Coastline 4015170 N052.01.28.701 W004.59.59.755
+REGIONNAME Park Air MAN (Land)
```

Comment:
Can we leave the name as Coastline? Think Chris' intention is to phase out this region, and have just spoken with Kieran as well and think it would be best just to use the A- regions. Thus, we don't really want people's displays breaking in the mean time if we're going to force them over to different regions anyway